crowdsourcing/viewsets/worker.py

```diff
@@ -47,7 +47,7 @@ class WorkerViewSet(viewsets.ModelViewSet):
     queryset = Worker.objects.all()
     serializer_class = WorkerSerializer
     lookup_field = 'profile'
-    permission_classes = [IsOwnerOrReadOnly]
+    # permission_classes = [IsOwnerOrReadOnly]
```

Comment:
ok, i see, then either this permission class needs to be updated OR the user can only update his own data